### DISCUSSION TITLE
MP3 output error

### DISCUSSION THREAD

**Discussion by mmssofter** [CarlGao4/Demucs-Gui#56] [Q&A] [answered]
Demucs-Gui works absolutely fine when outputting wav or flac, but if MP3 is selected for output, the separation process won't start. 

In the _demucs_gui_log.log_ the following entries are recorded whenever MP3 output is selected and the "Start Separation" button is clicked...

_2024-05-04 14:03:36,170 (shared.py) (Line 245) [DEBUG] : (switchEncoder) Set history ('encoder',) to 1
Traceback (most recent call last):
  File "GuiMain.py", line 1922, in startSeparation
AttributeError: 'ExpandingQPlainTextEdit' object has no attribute 'text'_

Notes:
The default MP3 command line is displayed...     _ffmpeg -y -v level+warning -i - -c:a libmp3lame -b:a 320k {output}_ 
FFMpeg appears to be recognised/used by Demucs_Gui, according to the demucs log.
MP3 input files can be successfully separated into WAV or FLAC stems.

Any help/advice would be appreciated.

**Comment by CarlGao4** [comment]
Found issue. Will be fixed in the next release.

**Reply by mmssofter** [reply]
Excellent, thank you.

**Comment by CarlGao4** [comment, marked as answer]
Please try 1.2b1

**Reply by mmssofter** [reply]
Many thanks, MP3 output is now fully working in 1.2b1
